site/content/en/docs/Reference/API/Kustomization File/labels.md

```diff
@@ -6,5 +6,35 @@ weight: 10
 description: >
     Add labels and optionally selectors to all resources.
 ---
+`apiVersion: kustomize.config.k8s.io/v1beta1`
 
-The Tasks section contains examples of how to use [`labels`](/docs/tasks/labels_and_annotations/).
+See the [Tasks section] for examples of how to use `labels`.
+
+### labels
+Adds labels and optionally selectors to all resources.
+
+* **labels** ([]Label)
+
+    List of labels and label selector options.
+
+    _Label holds labels to add to resources and options for customizing how those labels are applied, potentially using selectors and template metadata._
+
+    * **pairs** (map[string]string)
+
+        Map of labels that the transformer will add to resources.
+
+    * **includeSelectors** (bool), optional
+
+        IncludeSelectors indicates whether the transformer should include the fieldSpecs for selectors. Custom fieldSpec specified by `fields` will be merged with builtin fieldSpecs if this is true. Defaults to false.
+
+    * **includeTemplates** (bool), optional
+
+        IncludeTemplates indicates whether the transformer should include the `spec/template/metadata` fieldSpec. Custom fieldSpecs specified by `fields` will be merged with the `spec/template/metadata` fieldSpec if this is true. If IncludeSelectors is true, IncludeTemplates is not needed. Defaults to false.
+
+    * **fieldSpecs** (\[\][FieldSpec]({{< relref "../Common%20Definitions/FieldSpec.md" >}})), optional
```

Comment:
Is the goal here to list the field how it's called in code, or how a user would use it?

If it's the latter, this should be `fields` instead:
```suggestion
    * **fields** (\[\][FieldSpec]({{< relref "../Common%20Definitions/FieldSpec.md" >}})), optional
```
as specified in the Go field tags: https://github.com/kubernetes-sigs/kustomize/blob/3e69c2e36a5b9d7959c66f7ed39d54dec67dd8ae/api/types/labels.go#L19

Reply:
I believe the goal is to represent how a user can use this field in a kustomzation file. I updated this to `fields`. Thanks for catching this!